tiledb/tests/test_libtiledb.py

```diff
@@ -3769,17 +3775,12 @@ def test_tiledb_py_0_6_anon_attr(self):
             self.assertEqual(qres["d"], 0)
 
 
+pytest.mark.skipif(sys.platform == "win32", reason="Only run MemoryTest on linux")
```

Comment:
This should be a `MemoryTest` @decorator, right?

Reply:
yep, missed the extra \n. thanks